beancount_import/matching.py

```diff
@@ -772,6 +880,11 @@ def add_subset(account, currency, subset, check_zero=True):
                 2, min(len(posting_list) + 1, max_subset_size + 1)):
             for subset in itertools.combinations(posting_list, subset_size):
                 add_subset(account, currency, subset)
+
```

Comment:
We already loop over subset_size just above here, so it would better to re-order the looping above so that we can avoid spending the time generating subsets that we will just throw away anyway.

Reply:
I have addressed this in my current code.